configs/config_sandbox/esrf16_altmodels_study2/train-2dmodels-fpn-efficientnetb0.yaml

```diff
@@ -0,0 +1,19 @@
+train_config:
+  model_id_prefix: 'segmentation-model-fpn-efficientnetb0-study2'
+  dataset_id: 'dataset-esrf16_study2_CropCase5_TrnValCaseB'
+  segmentation_model:
+    model_name: 'FPN'
```

Comment:
major addition we need to everything below is options for each as comment next to selection (see dataset config for example of this). need to ref `models.py` to see what options are currently supported. the template needs to list all options where fixed options are possible